webpack.config.babel.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+const webpack = require('webpack');
+const nodeExternals = require('webpack-node-externals');
+const BabiliPlugin = require('babili-webpack-plugin');
+const DotenvPlugin = require('dotenv-webpack');
+const path = require('path');
+
+const config: webpack.Configuration = {
```

Comment:
Typescript, so it might not work 100%. It's there mostly for documentation though. You can pull and try it out on vscode lol